features/step_definitions/outcome.rb

```diff
@@ -71,4 +71,9 @@
 
 Alors("{string} est le document affiché à l'écran") do |text|
   expect(find('.runningHead')).to have_content text
+end
+
+Alors('la vidéo du document principal {string} se lance de {string} secondes à {string} secondes') do |url, start, ending|
+  path = url + '?start='+ start +'&end=' + ending + '&autoplay=1&mute=1'
+  expect(find('iframe')['src']).to include(path)
```

Comment:
This step doesn't pass on GitHub actions. Did it pass on your laptop?

If I remember right, members of your team (@AVCharpentier and @marina-pera) already faced the same problem and found a solution by using `within` (see https://github.com/Hypertopic/HyperGlosae/blob/main/features/step_definitions/outcome.rb#L62-L66).

Reply:
The test works properly on our computers without using within_frame.
![image](https://github.com/Hypertopic/HyperGlosae/assets/95986857/e3a7c97e-f2a0-4b9f-8325-c79407bb78a9)

With within_frame :
![image](https://github.com/Hypertopic/HyperGlosae/assets/95986857/0f8eaa9d-ff61-4665-8a75-60912f511d3c)
This is normal that this second version does not work on our computers and on GitHub. Here we change the context of research. We search in the content of the iframe, that is to say on the youtube page and not on the Hyperglossae site.

Note that the youtube page does not include the beginning and end of the extract. The management of video playback is processed by a javascript script and not stored in an HTML element, so complicated to make a classic find to perform a search in an HTML element. 
![image](https://github.com/Hypertopic/HyperGlosae/assets/95986857/99267c7f-f412-447c-b442-0f0b239da7bc)

Then I modified the test to search JS scripts.
![image](https://github.com/Hypertopic/HyperGlosae/assets/95986857/f62e7d05-a7af-458c-a800-4ed759eb6588)

I am not sure that relying on the youtube page is the best solution because Youtube changes their structure very regularly which could lead to the test not working later.

I hope this modification will fix github issue

Reply:
Obviously the tests are still failing on GitHub. I wonder if the problem is more of a commit order nature. Maybe it runs the tests commit by commit without worrying about next commits.

I wonder this because the tests are essentially based on the commit of samples. If the samples are not good, tests based on them will not work.

The failed test error would come from there?